core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2.scala

```diff
@@ -59,27 +60,42 @@ trait BillingApiServiceV2 extends UserInfoDirectives {
     requireUserInfo(Option(otelContext)) { userInfo =>
       val ctx = RawlsRequestContext(userInfo, Option(otelContext))
       pathPrefix("billing" / "v2") {
-        pathPrefix("spendReport") {
-          pathEndOrSingleSlash {
-            get {
-              parameters(
-                "startDate".as[DateTime],
-                "endDate".as[DateTime],
-                "pageSize".as[Int],
-                "offset".as[Int]
-              ) { (startDate, endDate, pageSize, offset) =>
+        pathPrefix("id") {
+          pathPrefix(Segment) { id =>
+            pathEnd {
```

Comment:
nit: `pathPrefix(…) { pathEnd {}}` is redundant; you can just do:

```suggestion
          path(Segment) { id =>
```

Reply:
Fixed: https://github.com/broadinstitute/rawls/pull/3205/commits/a150b008292b4d4bc5aa972d9fe8b0e238c552a6